docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: "3.8"
+
+services:
+  frontend:
+    build: ./front-end
+    ports:
+      - 4200:80
+    depends_on:
+      - backend
+
+  backend:
+    build: ./back-end
+    ports:
+      - 3000:3000
+    environment:
+      DB_URL: mongodb://127.0.0.1:27017/todo/tasks
+    depends_on:
+      - database
+
+  database:
```

Comment:
a good practice is to name your containers in a way that they are distinguishable from other containers. If you have another home project with some other name but with the same service name, that's quite confusing. I'd suggest giving every service a `todo-` prefix

Reply:
Docker Desktop automaticly does that. And I think it's the same way in Docker CLI, too.

![image](https://github.com/bencefarkas200/to-do/assets/148462547/28ec91ba-e96c-4987-b950-10f23b0cb89b)

Reply:
Indeed!

Can confirm that it works the same with the CLI too.